chia/types/spend_bundle.py

```diff
@@ -77,7 +78,7 @@ def estimate_fees(spend_bundle: SpendBundle) -> int:
     for cs in spend_bundle.coin_spends:
         removed_amount += cs.coin.amount
         coins, cost = compute_additions_with_cost(cs, max_cost=max_cost)
-        max_cost -= cost
+        max_cost = uint64(max_cost - cost)
         if max_cost < 0:
             raise ValidationError(Err.BLOCK_COST_EXCEEDS_MAX, "estimate_fees() for SpendBundle")
```

Comment:
This is losing the type and message of the raised exception since `uint64()` would raise.